examples/stats/prometheus/main.go

```diff
@@ -39,51 +39,28 @@ func main() {
 
 	// Create measures. The program will record measures for the size of
 	// processed videos and the number of videos marked as spam.
-	videoCount, err := stats.Int64("my.org/measures/video_count", "number of processed videos", "")
-	if err != nil {
-		log.Fatalf("Video count measure not created: %v", err)
-	}
-
-	// 1. Create view to see the number of processed videos cumulatively.
-	viewCount, err := view.New(
-		"video_count",
-		"number of videos processed over time",
-		nil,
-		videoCount,
-		view.CountAggregation{},
-	)
-	if err != nil {
-		log.Fatalf("Cannot create view: %v", err)
-	}
+	videoCount, _ := stats.Int64("my.org/measures/video_count", "number of processed videos", "")
```

Comment:
As I had mentioned offline, examples are our face of style and in my experience I've found
lots of users copied and pasted the examples so let's do some error handling here to
avoid future bug reports in which users can tell where the nil dereference came from
yet handling the error could have helped them.

Reply:
ack